includes/class-wc-taxjar-transaction-sync.php

```diff
@@ -29,29 +29,27 @@ public function __construct( $integration ) {
 	 * Add actions and filters
 	 */
 	public function init() {
-		if ( apply_filters( 'taxjar_enabled', isset( $this->taxjar_integration->settings['enabled'] ) && 'yes' == $this->taxjar_integration->settings['enabled'] ) ) {
+		if ( isset( $this->taxjar_integration->settings['taxjar_download'] ) && 'yes' == $this->taxjar_integration->settings['taxjar_download'] ) {
 			add_action( 'init', array( __CLASS__, 'schedule_process_queue' ) );
 			add_action( self::PROCESS_QUEUE_HOOK, array( $this, 'process_queue' ) );
 
```

Comment:
@dallendalton Can you elaborate on why these two actions were moved?  It looks like they used to run even if 
'taxjar_download' was disabled.

Reply:
@ryanreeves-taxjar - Good catch. Originally I thought that those actions were unnecessary unless transaction sync was enabled. However, your comment made me go back and second check and I realized there were some edge cases when customers would not be able to sync to TaxJar if I did that change. Customers usually sync synchronously so the sync queue isn't necessary, but there are some edge cases that utilize the sync queue.

I have updated the PR with the correct logic. Thanks!